SIAB/spillage/lcao_wfc_analysis.py

```
@@ -46,7 +47,135 @@ def _wll(C, S, natom, nzeta):
+
+    ntyp = len(nzeta)
+
+    assert pop_view in ['decompose', 'reduce']
+
+    nao, nbands_max = C.shape
+    nbands = nbands_max if nbands == 'all' else nbands
+    assert nbands <= nbands_max, 'nbands selected is larger than the total nbands'
+
+    lin2comp = _lin2comp(natom, nzeta)
+    comp2lin = sorted([((it, l, m, iz, ia), i)
+                        for i, (it, ia, l, iz, m) in enumerate(lin2comp)])
```

Ruff

Ambiguous variable name: `l`

(E741)